### DISCUSSION TITLE
In Combat Conditional?

### DISCUSSION THREAD

**Discussion by Richy1111** [AscensionGameDev/Intersect-Engine#1487] [Ideas]
I was wondering if we can maybe have an 'In Combat' conditional check for events. I don't want to lock a player inside a PvP Arena but I want to make sure they can't leave that Arena while in combat either?
Thanks.

**Comment by WeylonSantana** [comment]
I think this can be easily implemented by returning if options.combatTimer + current time > current time, or something like

**Comment by da-rulez** [comment]
Im in favour of this one

**Comment by PyroTech03** [comment]
Is it possible to merge these? https://github.com/AscensionGameDev/Intersect-Engine/discussions/1930

**Comment by WeylonSantana** [comment]
solved in https://github.com/AscensionGameDev/Intersect-Engine/pull/2653#pullrequestreview-2675409907
